Account/FirefoxAccountConfiguration.swift

```diff
@@ -120,7 +123,7 @@ public struct StageFirefoxAccountConfiguration: FirefoxAccountConfiguration {
 }
 
 public struct ProductionFirefoxAccountConfiguration: FirefoxAccountConfiguration {
-    public init() {
+    public init(prefs: Prefs? = nil) {
```

Comment:
This change isn't needed if you remove the constructor from the `FirefoxAccountConfiguration`.